authapi/views.py

```diff
@@ -158,16 +158,52 @@ class TeamPermissionViewSet(
     '''Nested viewset to add and remove permissions from teams.'''
     queryset = SeedPermission.objects.all()
     serializer_class = PermissionSerializer
+    permission_classes = (permissions.TeamPermissionPermission,)
 
-    def create(self, request, parent_lookup_seedteam=None, **kwargs):
+    def check_team_permissions(self, request, teamid, orgid=None):
+        old_method = request.method
+        request.method = 'GET'
```

Comment:
I see. Just worried that when anything raises an exception before we restore the method, any further processing that might happen for the request (for whatever catches the exception) will assume we have a `GET`, which could let to some obscure errors.

Two thoughts:
  - Could we update a copy of the request instead?
  - Wondering if this method would make more sense on`TeamPermissionPermission` than the view